nexus_transporter/include/nexus_transporter/Transporter.hpp

```diff
@@ -58,18 +60,22 @@ class Transporter
   /// Return true if the transporter is configured and ready.
   virtual bool ready() const = 0;
 
-  /// Receive an itinerary for a destination
+  /// Request an itinerary for a list of destinations. This call should be
+  /// non-blocking.
   ///
   /// \param[in] job_id
   /// An id for this request.
   ///
   /// \param[in] destinations
   /// A list of destinations.
-  /// \return A nullopt is returned if the destinations is not valid.
+  ///
+  /// \param[in] completed_cb
+  /// A callback to execute when an itinerary query has been completed.
   // TODO(YV): Consider creating a separate class for destination
```

Comment:
Unrelated but we can remove this TODO.

Reply:
https://github.com/osrf/nexus/pull/89/commits/67b242227cba288ce15199f38b1584d82c234558